integration-tests/publish.go

```diff
@@ -0,0 +1,105 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"net/http"
+	"github.com/CESARBR/knot-cloud-storage/pkg/data"
+	"github.com/CESARBR/knot-cloud-storage/pkg/entities"
+	"github.com/CESARBR/knot-cloud-storage/pkg/logging"
+	"github.com/google/go-querystring/query"
+	"github.com/streadway/amqp"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"time"
+)
+
+func sendRequest(svc *requestInfo) (*http.Response, error) {
```

Comment:
In this file, you have HTTP and AMQP code together. Two options: rename this file to something more generic or separate them in two different files.

Reply:
Done. I put the HTTP code in the `http.go`file